config/router/src/common-bases.ts

```diff
@@ -0,0 +1,244 @@
+import { ChainId } from '@sushiswap/chain'
+import {
+  ARB,
+  DAI,
+  FRAX,
+  GNO,
+  MATIC,
+  MIM,
+  Native,
+  SUSHI,
+  Token,
+  USDC,
+  USDT,
+  WBTC,
+  WETH9,
+  WNATIVE,
+} from '@sushiswap/currency'
+
+export const COMMON_BASES = {
```

Comment:
Why not expand Common Bases to include more tokens on networks with fewer fees? Namely Sushi and XSushi. This should increase volume and trading fees that the tokens earn.